DS/DS2/MainStackImplementation.java

```diff
@@ -0,0 +1,77 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+//class used to define a stack and its operation
+class Node {
```

Comment:
ACTION: If node class doesn't contain anything then what is the point of creating it, If you just wanted to use it for abstraction then it should be interface not class.